### PR TITLE
Add new Autocomplete component

### PR DESCRIPTION
**Pull request by DanielaBeCause**

### CODE REVIEW

File: src/Autocomplete/Autocomplete.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+})
+
+export type TAutocomplete = TPlaywright & {
+  input: ReactElement
+  results: ReactElement | ReactElement[]
+
+  open: boolean
+  setOpen: (open: boolean) => void
+
+  colorPopOverBackground: EColor
```
Comment (cllpse) [inline src/Autocomplete/Autocomplete.tsx:24]: Just colorBackground will suffice here

File: src/Autocomplete/Autocomplete.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        alignment={EAlignment.Start}
+        trigger={input}
+        background={
+          <BackgroundCard
+            colorBackground={[colorPopOverBackground, 700]}
+            shadow={EShadow.Medium}
+            borderRadius={ESize.Tiny}
+          />
+        }
+        layout={<LayoutSingle omitPadding direction={EDirection.Vertical} content={results} />}
```
Comment (cllpse) [inline src/Autocomplete/Autocomplete.tsx:44]: Use <LayoutContextMenu />

File: src/OverflowContainer/OverflowContainer.tsx [hunk 3]
```diff
@@ -78,6 +80,7 @@ export const OverflowContainer = ({
     maxHeight={maxHeight}
     hug={hug}
     data-playwright-testid={playwrightTestId}
+    className={className}
```
Comment (cllpse) [inline src/OverflowContainer/OverflowContainer.tsx:83]: I have no better solution other than adding className?: string to TLayoutBase, and incorporating className on all of our components.

**Review by cllpse — APPROVED** [review]

File: src/OverflowContainer/OverflowContainer.tsx (hunk 3 above)
Comment (DanielaBeCause) [reply src/OverflowContainer/OverflowContainer.tsx:83]: but the `OverflowContainer` which is the component that needs className in my case, does not extend the `TLayoutBase` interface.

Comment (cllpse) [reply src/OverflowContainer/OverflowContainer.tsx:83]: Right. Screw it then 🗡️ 😄

File: src/Autocomplete/Autocomplete.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import { Popover } from "@new/Popover/Popover"
+import { TPlaywright } from "@new/TPlaywright"
+import { forwardRef, PropsWithChildren, ReactElement } from "react"
+
+const Container = styled.div({
+  display: "flex",
+})
+
+export type TAutocomplete = TPlaywright & {
+  input: ReactElement
```
[the remark this answers is not in the document]
Comment (DanielaBeCause) [reply src/Autocomplete/Autocomplete.tsx:18]: It should also support InputButton

[the remark this answers is not in the document]
Comment (DanielaBeCause) [reply src/Autocomplete/Autocomplete.tsx:18]: image

**Pull request merged**